packages/jazz-browser-prosemirror/src/schema.ts

```diff
@@ -0,0 +1,7 @@
+import { CoRichText } from "jazz-tools";
+
+/**
+ * Extends CoRichText to represent a collaborative rich text document.
+ * Used as the underlying data structure for the ProseMirror editor.
+ */
+export class Document extends CoRichText {}
```

Comment:
Sorry, I'm not sure why I introduced this alias in my original branch - we don't need it and everything would be clearer if we just dealt with CoRichText everywhere. Please may you get rid of it @bensleveritt ?

Reply:
Removed!